.vscode/extensions.json

```diff
@@ -2,7 +2,7 @@
   "recommendations": [
     "nrwl.angular-console",
     "angular.ng-template",
-    "ms-vscode.vscode-typescript-tslint-plugin",
+    "dbaeumer.vscode-eslint",
```

Comment:
can we trust him? 😃

Reply:
yep its official microsoft version
https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint